bedrock/settings/static_media.py

```diff
@@ -1165,6 +1165,14 @@
         ),
         'output_filename': 'js/firefox_developer_firstrun-bundle.js',
     },
+    # will be needed when videos are ready (shortly after 6/2?)
```

Comment:
No, wait. This bundle is not needed at the moment.

Videos did not come in for the features promoted on `/firefox/40.0a2/whatsnew/`, so we don't need any page-specific JS running. The videos will likely come in at some point, and when they do, we'll have this bundle ready.